salt/beacons/avahi_announce.py

```diff
@@ -12,8 +12,9 @@
 
 '''
 # Import Python libs
-from __future__ import absolute_import
+from __future__ import absolute_import, unicode_literals
 import logging
+import six
```

Comment:
The `six` import should be from salt's ext lib as we ship our own version with Salt:
```
from salt.ext import six
```